tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -33,7 +33,7 @@
 
 DISABLE_RE_STAKE_FIELD = 3
 
-DISABLED_FIELD = 5
+DISABLED_FIELD = 4
```

Comment:
I'm sure it's supposed to be obvious, but help me catch up: why did this change?

Reply:
Changed index of `disabled` slot in PolicyManager.Policy struct, because `firstPartialReward` was removed

Reply:
Ahh, roger, got it.